app/jobs/reports/monthly_key_metrics_report.rb

```diff
@@ -32,19 +53,29 @@ def emails
       emails
     end
 
-    def monthly_key_metrics_report_array
-      csv_array = []
+    def account_reuse_queries
+      @account_reuse_queries ||= Reporting::AccountReuseAndTotalIdentitiesReport.new(report_date)
+    end
 
-      account_reuse_report_csv.each do |row|
-        csv_array << row
-      end
+    def upload_to_s3(report_body, report_name: nil)
+      _latest, path = generate_s3_paths(REPORT_NAME, 'csv', subname: report_name, now: report_date)
```

Comment:
so we're not using `_latest` here which means we're only writing the timestamped copy of the file, not the `latest` reference, is that intentional?